pages/docs/index.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import Button from "../../components/Button";
+import Link from "next/link";
+import MainAppBar from '../../views/MainAppBar';
+
+const docs = [
+	{
+		heading: "Github",
+		flag: true,
+		content: [
+			{ subHeading: "Sign up with github", article: "Sign in using GitHub" },
+			{ subHeading: "Install GitHub App", article: <>Install <Link href="https://github.com/apps/repoprofiler" className="text-blue-500">Repo Profiler Github App</Link> from Github Marketplace in your org/personal account. Make sure you have the permissions required to install the app.</> },
+			{
+				subHeading: "Setup GitHub Action",
+				article:
+					<div>
+						<pre className="bg-gray-100 rounded-md p-3 mb-4 font-mono whitespace-pre-wrap">
+							<code>
+								{`on:
+	repository_dispatch:
+		types: repo_profile_execution
+jobs:
+	profile:
+		runs-on: ubuntu-22.04
+		steps:
+			- name: Checkout
+			uses: actions/checkout@v3
+			with:
+				fetch-depth: 0
+			- name: Repository Profiler
+			uses: Alokit-Innovations/repo-profiler@main`}
```

Comment:
Please copy from [devprofiler-cloudrun](https://github.com/Alokit-Innovations/devprofiler-cloudrun/blob/main/.github/workflows/repo-profiler.yml) for correct formatting